src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -251,9 +254,9 @@ private Request make(final byte[] boundary) throws IOException {
             }
             return new RqWithHeader(
                 new RqLive(
-                    new CapInputStream(
```

Comment:
@yegor256 CapInputStream is not needed for files, the stream knows file length without CapInputStream